File: SPSurvey.Core/SDK/ApplicationUserItem.vb
Public Class ApplicationUserItem

    Property ApplicationUserID As Integer 
    Property UserLogin As String
    Property AccountNM As String 
    Property FirstNM As String 
    Property LastNM As String 
    Property eMailAddress As String 
    Property CommentDS As String 
    Property CompanyID As Integer
    Property CompanyNM As String 
    Property SupervisorAccountNM As String 
    Property LastLoginDT As DateTime 
    Property LastLoginLocation As String 
    Property SurveyResponseList As New List(Of SurveyResponseItem) 
    Property ApplicationUserRoleList As New List(Of ApplicationUserRoleItem) 
    Property SurveyResponseCount As Integer
    Property ApplicatonUserRoleCount As Integer
    Property ModifiedDT As DateTime
    Property ModifiedID As Integer
    Property HasMessages As Boolean
    Property MessageCount As Integer
    Property UserRoleID As Integer
    Property UserRoleName As String
    Property Messages As New List(Of SiteMessageItem)
    Property Properties As New List(Of UserAppPropertyItem)
    Property DisplayName As String
    Property UserKey As GUID 
    Property VerifyCode As String 
    Property EmailVerified As Boolean
End Class

File: ControlOrigins.Survey.Common/SDK/ApplicationUserItem.cs
using ControlOrigins.Survey.Common.SDK.SurveyResponse;
using System;
using System.Collections.Generic;

namespace ControlOrigins.Survey.Common.SDK
{
    public class ApplicationUserItem
    {
        public int ApplicationUserID { get; set; }
        public string UserLogin { get; set; }
        public string AccountNM { get; set; }
        public string FirstNM { get; set; }
        public string LastNM { get; set; }
        public string EMailAddress { get; set; }
        public string CommentDS { get; set; }
        public int CompanyID { get; set; }
        public string CompanyNM { get; set; }
        public string SupervisorAccountNM { get; set; }
        public DateTime LastLoginDT { get; set; }
        public string LastLoginLocation { get; set; }
        public List<SurveyResponseItem> SurveyResponseList { get; set; } = new List<SurveyResponseItem>();
        public List<ApplicationUserRoleItem> ApplicationUserRoleList { get; set; } = new List<ApplicationUserRoleItem>();
        public int SurveyResponseCount { get; set; }
        public int ApplicationUserRoleCount { get; set; }
        public DateTime ModifiedDT { get; set; }
        public int ModifiedID { get; set; }
        public bool HasMessages { get; set; }
        public int MessageCount { get; set; }
        public int UserRoleID { get; set; }
        public string UserRoleName { get; set; }
        public List<SiteMessageItem> Messages { get; set; } = new List<SiteMessageItem>();
        public List<UserAppPropertyItem> Properties { get; set; } = new List<UserAppPropertyItem>();
        public string DisplayName { get; set; }
        public Guid UserKey { get; set; }
        public string VerifyCode { get; set; }
        public bool EmailVerified { get; set; }
    }
}